node/full.go

```diff
@@ -147,6 +150,16 @@ func (qm *QitmeerFull) RegisterAccountService(cfg *config.Config) error {
 	return nil
 }
 
+func (qm *QitmeerFull) RegisterWalletService(cfg *config.Config, evm *meer.MeerChain, _am *acct.AccountManager, _tm *tx.TxManager, events *event.Feed) error {
```

Comment:
same reason here. why component depends on each others. what's the relationship with each others. if the service has to be registered in the fixed order (failed if the order is changed), please reconsider your design.